ddpui/dbt_automation/operations/flattenjson.py

```
@@ -30,10 +31,18 @@ def flattenjson_dbt_sql(
     json_column = config["json_column"]
     json_columns_to_copy = config["json_columns_to_copy"]
 
-    if source_columns == "*":
-        dbt_code = "SELECT *\n"
-    else:
-        dbt_code = f"SELECT {', '.join([quote_columnname(col, warehouse.name) for col in source_columns])}\n"
+    # Determine the source columns to be selected
+    if config["source_columns"] == "*":
+        raise HttpError(
```

could you raise an Exception instead of an HttpError

this is very deep in the code, far disconnected from the http request